helpers/albanode.py

```diff
@@ -13,32 +13,30 @@
 #
 # Open vStorage is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY of any kind.
+from ci.scenario_helpers.ci_constants import CIConstants
```

Comment:
We might have glossed over something completely here.
The idea was that the automation library would be completely independent from anything (currently not the case as lookups are still happening by using the DAL instead of the API, https://github.com/openvstorage/automation-lib/issues/57 would remove this dependency)
Now with these imports, the integration tests have to be installed together with the automation_lib in order for it to work. We do not want this.
Instead I propose the following:

- Move the CIConstants api part to the automation lib (let it read from the same file)
- Strip the integration test stuff from CIConstants in automation lib
- Create a new CIConstants inside the integration tests which inherit the code from api_lib
- Apply the required stuff for integration tests in the new class